src/app/components/MediaPlayer/AudioLoader/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { node } from 'prop-types';
+import styled from '@emotion/styled';
+import { keyframes } from '@emotion/react';
+
+const Wrapper = styled.div`
+  position: relative;
+`;
+
+const Overlay = styled.div`
+  .no-js & {
+    display: none;
+  }
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+`;
+
+const spin = keyframes`
+0% {
+  transform: rotate(0deg);
+}
+100% {
+  transform: rotate(360deg);
+}
+`;
+
+const Spinner = styled.div`
+  width: 5rem;
+  height: 5rem;
+  border-radius: 50%;
+  border: 0.3rem solid rgba(0, 0, 0, 0.2);
+  border-left-color: rgba(0, 0, 0, 0.8);
```

Comment:
does this change with colour preferences in FF?

Reply:
Nope - what colour should it be?  FF options are just to change the "text colour" and the "background colour" - guessing it should follow the text colour/

Reply:
Yeh I think text colour 👍

Reply:
Had to move this to an SVG to support this - don't think it's possible to do it with CSS as firefox just takes over all colours I could find (borders, backgrounds, insets, outlines)